unlock-app/src/components/interface/locks/CheckoutUrl/index.tsx

```diff
@@ -323,6 +337,20 @@ export const CheckoutUrlPage = () => {
   const hasSelectedConfig =
     configuration === 'existing' && checkoutConfig?.id !== undefined
 
+  /**
+   * Save checkout config when fields have changed, This is done with delays invoking a function until after wait milliseconds have passed
+   * to avoid calling the endpoint multiple times.
+   */
+  const [_isReady] = useDebounce(
+    () => {
+      onConfigSave()
+    },
+    5000, // 5 seconds of delay after edit's to trigger auto-save
```

Comment:
This seems large. If the user is done, they make click thru to the next step in their journey and this would not be saved.
Any reason why you want 5 seconds?